config/tcs_config.toml

```diff
@@ -30,80 +30,15 @@ LogFile  = "__screen__"
 # Set this flag to 1 for sync workorder execution.Else set it to 0
 sync_workload_execution = 1
 
-# -------------------------------------------------------------------
-# EnclaveManager -- configuration of Intel SGX Enclave EnclaveManager
-# -------------------------------------------------------------------
-[EnclaveManager]
-# configurations for asynchronous work load execution
-sleep_interval = "10"
-# configurations for synchronous Work load execution
-# zmq_port is the port used for zmq socket communication
-# Port for socket interaction with listener
-zmq_port = '5555'
```

Comment:
Why did we remove the zmq_port? Enclave manager uses it right?

Reply:
Refactored to singleton_enclave_config.toml.